<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Account_Form</class>
 <widget class="QMainWindow" name="Edit_Account_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(G:/Final1/Image_EditAccount.jpg) ;
background-repeat: no-repeat;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Matura MT Script Capitals&quot;;
background-color:black;
color:white;</string>
    </property>
    <property name="text">
     <string>Edit Account</string>
    </property>
   </widget>
   <widget class="QComboBox" name="List">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Password">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Account_Type">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Username">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Username_Label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="ContactNo_Label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Contact No:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password_Label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Address">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="ContactNo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Address_Label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name_Label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Account_Type_Label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Account Type: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getAddress">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="New_Name_Label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="New_CNIC_Label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>CNIC: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getUsername">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="New_ContactNo_Label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Contact No:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getContactNo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="getCNIC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="getName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="New_Address_Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getPassword">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="New_Username_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Username: </string>
    </property>
   </widget>
   <widget class="QLabel" name="New_Password_Label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Password: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>560</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0);
font: 20pt &quot;Bauhaus 93&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
